src/odemis/gui/comp/stream_panel.py

```diff
@@ -140,9 +140,9 @@ def __init__(self, parent, wid=wx.ID_ANY, pos=wx.DefaultPosition, size=wx.Defaul
     def _add_remove_btn(self):
         """ Add a button for stream removal """
         btn_rem = buttons.ImageButton(self,
-                                      bitmap=img.getBitmap("icon/ico_rem_str.png"),
```

Comment:
If add stream button is a `+` shouldn't remove stream button be a `-`?

Reply:
Users are assuming that the - is minimise, because that's the OS symbol, and closing when they mean to minimise. So the idea is to match the OS where X is close/ remove.